tests/clone_test.go

```diff
@@ -543,17 +543,26 @@ var _ = Describe("VirtualMachineClone Tests", Serial, func() {
 						}
 					})
 
-					DescribeTable("should create new ControllerRevisions for cloned VM", Label("instancetype", "clone"), func(runStrategy virtv1.VirtualMachineRunStrategy) {
+					DescribeTable("should create new ControllerRevisions for cloned VM", Label("instancetype", "clone"), func(isStopped bool) {
+						runStrategy := virtv1.RunStrategyOnce
```

Comment:
I think its better to change the behavior only in case of halted and wffc storage.
i.e if runStrategy == virtv1.RunStrategyHalted && libstorage.IsStorageClassBindingModeWaitForFirstConsumer(snapshotStorageClass) 
In that case start the vm with run strategy `Always` and then stop it as you did.
And move the comment you put there to before this if

Reply:
Agree. Fixed.